api/src/test/java/org/pmiops/workbench/test/FileUtils.java

```diff
@@ -0,0 +1,18 @@
+package org.pmiops.workbench.test;
+
+import com.google.common.io.CharStreams;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.Charset;
+
+public class FileUtils {
+
+  public static String readFileContents(String path) throws Exception {
```

Comment:
If you're not using this anymore, I think you can get rid of it.

Reply:
Removed.